### PR TITLE
[ATL-607] Allow filtering invoices by subscription

### PR DESCRIPTION
**Pull request by rainbow-alex**

### CODE REVIEW

**Review by BrechtBonte — DISMISSED** [review]
:heavy_check_mark: Looks good and renders correctly
image

**Review by JordyDeGroeve — DISMISSED** [review]

**Review by Jdww — DISMISSED** [review]

**Comment by joaoinacio** [comment]
It should also update the changelog (non-breaking changes)

**Review by MathiasDL — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -397,6 +397,10 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
+### February 2022
```
Comment (Jdww) [inline apiary.apib:400]: PP: We've been using Latest for the current month.

**Review by Jdww — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (JordyDeGroeve) [reply apiary.apib:400]: We stopped doing that recently, right? 🤔  To avoid we need to update it every month?

**Review by JordyDeGroeve — DISMISSED** [review]
Conflicts due to my recent merge. Sorry!
Changes look fine.

**Review by JordyDeGroeve — APPROVED** [review]

**Review by Jdww — APPROVED** [review]

**Review by MathiasDL — APPROVED** [review]

**Pull request merged**